src/test-data/sample-ingredient-data.js

```diff
@@ -0,0 +1,59 @@
+let ingredientsData = [
+  {
+    "id": 20081,
+    "name": "wheat flour",
+    "estimatedCostInCents": 142
+  },
+  {
+    "id": 18372,
+    "name": "bicarbonate of soda",
+    "estimatedCostInCents": 582
+  },
+  {
+    "id": 1123,
+    "name": "eggs",
+    "estimatedCostInCents": 472
+  },
+  {
+    "id": 19335,
+    "name": "sucrose",
+    "estimatedCostInCents": 902
+  },
+  {
+    "id": 19206,
+    "name": "instant vanilla pudding",
+    "estimatedCostInCents": 660
+  },
+  {
+    "id": 19334,
+    "name": "brown sugar",
+    "estimatedCostInCents": 559
+  },
+  {
+    "id": 2047,
+    "name": "salt",
+    "estimatedCostInCents": 280
+  },
+  {
+    "id": 1012047,
+    "name": "fine sea salt",
+    "estimatedCostInCents": 528
+  },
+  {
+    "id": 10019903,
+    "name": "semi sweet chips",
+    "estimatedCostInCents": 253
+  },
+  {
+    "id": 1145,
+    "name": "unsalted butter",
+    "estimatedCostInCents": 617
+  },
+  {
+    "id": 2050,
+    "name": "vanilla",
+    "estimatedCostInCents": 926
+  }
+]
+
+export default ingredientsData;
```

Comment:
Way to cut down on the unnecessary amount of fake data needed for the tests here